include/cppkafka/kafka_handle_base.h

```diff
@@ -262,6 +262,34 @@ class CPPKAFKA_API KafkaHandleBase {
      */
     int get_out_queue_length() const;
 
+#if RD_KAFKA_VERSION >= RD_KAFKA_DESTROY_FLAGS_SUPPORT_VERSION
+    /**
+     * \brief Sets destroy_flags
+     *
+     * Destroy flags will be used for handle destruction.
+     *
+     * Single values: 0 (Default), RD_KAFKA_DESTROY_F_NO_CONSUMER_CLOSE
+     *
+     * Setting to RD_KAFKA_DESTROY_F_NO_CONSUMER_CLOSE prevents potential
```

Comment:
I would say that this should be describe like in librdkafka originally (or just see rdkafka) and this leaved as a note